cliquet/events.py

```diff
@@ -41,25 +41,47 @@ def __init__(self, action, timestamp, read_records, request):
 
 
 class ResourceChanged(_ResourceEvent):
-    """Triggered when a resource is changed.
+    """Triggered when a resource is being changed.
```

Comment:
From what I understand, this event is know deprecated? If so, we should issue a warning saying so.